app/workers/clean_up_course_options_with_no_study_mode.rb

```diff
@@ -0,0 +1,23 @@
+class CleanUpCourseOptionsWithNoStudyMode
+  include Sidekiq::Worker
+
+  def perform
+    CourseOption.where('study_mode = \'0\'').each do |nil_co|
+      if nil_co.application_choices.any?
+        equivalent_course_option = CourseOption.where(
+          course: nil_co.course,
+          site: nil_co.site,
+          study_mode: :full_time,
+        ).first
+        if equivalent_course_option
+          nil_co.application_choices.each do |application_choice|
+            application_choice.update!(course_option_id: equivalent_course_option.id)
+          end
+          nil_co.destroy!
+        end
```

Comment:
Do we need to flag some kind of warning if there is no equivalent?

Or just fix it up so that `nil_co.study_mode` is set to `full_time`

Reply:
I like this suggestion, if no duplicate record exists for this course option, then we set it to the original intended value (we have this information based on the raw db value, which can be '0' or '1').